src/tests/ftest/pool/pool_svc.py

```diff
@@ -57,49 +58,58 @@ def test_poolsvc(self):
         try:
             # initialize a python pool object then create the underlying
             # daos storage
-            self.pool = DaosPool(self.context)
-            self.pool.create(createmode, createuid, creategid,
-                             createsize, createsetid, None, None, createsvc[0])
-            self.pool.connect(1 << 1)
-
+            self.pool = TestPool(self.context,
+                                 dmg_bin_path=self.basepath + '/install/bin')
+            self.pool.get_params(self)
+            # Manually set TestPool members before calling create
+            self.pool.mode.value = createmode
+            self.pool.scm_size.value = createsize
+            self.pool.name.update(createsetid)
+            self.pool.svcn.update(createsvc[0])
+            self.pool.create()
```

Comment:
There should be a `try/except TestFail` on this call to account for an invalid svcn.

Reply:
Create a try/except TestFail block around this create.